src/app/data-explorer/querybuilder/querybuilder.component.spec.ts

```diff
@@ -47,6 +51,10 @@ describe('QueryBuilderComponent', () => {
           provide: TerminologyService,
           useValue: terminologyStub,
         },
+        {
+          provide: MdmResourcesConfiguration,
+          useValue: mdmResourcesConfiguration,
```

Comment:
Providing `MdmResourcesConfiguration`  as a value to the `TestBedModule` looks like it is trying to get the tests running in any way possible. I think the tests should be updated to reflect the new functionality of getting the descriptions instead.

Reply:
I have added a test to make sure that descriptions are mapped correctly.